README.md

```diff
@@ -14,7 +14,7 @@ This can done one by converting your pfx file to a base64 encoded string with th
 certutil -encode .\ssCertInfo.pfx .\ssCertInfo.base64.txt
 ```
 
-Once you run the command, you will need to ensure that only the base64 data is saved in your secret as seen here (truncated)
+Once you run the command, you will need to ensure that the data is saved in your secret as seen here (truncated)
```

Comment:
The example below is not truncated? Can you correct this line?

Reply:
Hi, do you mean that this line should remove the mentioning of `(truncated)` as: 

`Once you run the command, you will need to ensure that the data is saved in your secret as seen here`

because the user no longer need to truncate the `-----BEGIN CERTIFICATE-----` and `-----END CERTIFICATE-----` lines?

Reply:
Yup that's what I meant.